src/AppBundle/DependencyInjection/AppExtension.php

```diff
@@ -38,5 +38,10 @@ public function prepend(ContainerBuilder $container): void
             $container->prependExtensionConfig($prependConfig, $config);
             $container->addResource(new FileResource($configFile));
         }
+
+        $configFile = __DIR__ . '/../Resources/config/content_view.yml';
```

Comment:
Instead of duplicating this, why not just add the entry to `$prependConfigs`, so you'd have:

```
        $prependConfigs = [
            'content_view.yml' => 'ezpublish',
            'layouts/blocks.yml' => 'netgen_layouts',
            'layouts/block_view.yml' => 'netgen_layouts',
            'layouts/item_view.yml' => 'netgen_layouts',
        ];
```

Reply:
Ah, okay, I understand now, it's because of added `system` key.